CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Instead of adding this file, I think it's better to add a section `## Code of Conduct` to the `README.md` and link to the one in our welcoming/landing page repo here: https://github.com/elixir-cloud-aai/elixir-cloud-aai/blob/dev/CODE_OF_CONDUCT.md

Additional advantage is that we already have the contact email address filled out in that one